v2alpha1/api/swagger/swagger.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: HPE GreenLake Metal Client API
-  version: 1.3.8
+  version: 0.0.1
```

Comment:
Will this eventually be 2.x.y? Or, will we consider using git tags?

Reply:
its 0.0.1 of v2alpha1 swagger spec. I will discuss and update later whether to keep it the same as tags or introduce a check to make sure this value is updated whenever yaml files are changed.